index.js

```diff
@@ -40,34 +40,35 @@ exports.register = (server, options) => {
 
   // catch request errors/warnings/etc (default: only errors) and capture them with sentry
   server.events.on({ name: 'request', channels: opts.channels }, async (request, event) => {
-    // format a sentry event from the request and triggered event
-    const sentryEvent = await Sentry.Parsers.parseError(event.error);
-    Sentry.Handlers.parseRequest(sentryEvent, request.raw.req);
+    Sentry.withScope(scope => { // thus use a temp scope and re-assign it
+      scope.addEventProcessor(_sentryEvent => {
```

Comment:
Why did you rename this to _sentryEvent?

btw: looks like an awesome PR, thanks!

Reply:
I think @jardakotesovec used `_sentryEvent` as a temporary handle for parsing it two lines below.
The parsed `sentryEvent` variable is used as before (manipulate `sentryEvent.request.url`, `sentryEvent.level`, etc).